### PR TITLE
FIX: Css changes

### PR DESCRIPTION
**Pull request by Ahmedgagan**

### CODE REVIEW

**Comment by Ahmedgagan** [comment]
@jjaffeux

**Comment by Ahmedgagan** [comment]
@jjaffeux I think the CSS code is quite messed up, I am trying to make it simple in the next commit 🙂

**Comment by Ahmedgagan** [comment]
@jjaffeux I added a commit with simplified CSS code. do have a look when you get time😃

File: assets/stylesheets/common/discourse-reactions.scss [hunk 1]
```diff
@@ -340,3 +325,8 @@ nav.post-controls .actions button {
     height: 15px;
   }
 }
+
+nav.post-controls .actions button:focus {
```
Comment (jjaffeux) [inline assets/stylesheets/common/discourse-reactions.scss]: can we have something that impacts only reactions button here ?

File: assets/stylesheets/mobile/discourse-reactions.scss [hunk 2]
```diff
@@ -5,8 +5,6 @@
   }
 
   .discourse-reactions-counter {
-    &:hover {
-      cursor: pointer;
-    }
+    margin: 0px;
```
Comment (jjaffeux) [inline assets/stylesheets/mobile/discourse-reactions.scss]:
```suggestion
    margin: 0;
```

File: assets/stylesheets/common/discourse-reactions.scss (hunk 1 above)
Comment (Ahmedgagan) [reply assets/stylesheets/common/discourse-reactions.scss]: sure, ill make it more specific

Comment (Ahmedgagan) [reply assets/stylesheets/common/discourse-reactions.scss]: Added a commit with changes

File: assets/stylesheets/common/discourse-reactions.scss [hunk 3]
```diff
@@ -340,3 +325,8 @@ nav.post-controls .actions button {
     height: 15px;
   }
 }
+
+nav.post-controls .actions .discourse-reactions-actions button:focus {
```
Comment (jjaffeux) [inline assets/stylesheets/common/discourse-reactions.scss]: do we need "button" ?

Comment (Ahmedgagan) [reply assets/stylesheets/common/discourse-reactions.scss]: I think so, as `discourse-reactions-actions` is div in which button's focus causes the issue

Comment (jjaffeux) [reply assets/stylesheets/common/discourse-reactions.scss]: my point is we don't want to target html elements directly, it adds specificity

Comment (jjaffeux) [reply assets/stylesheets/common/discourse-reactions.scss]: I recommend you read this article: https://css-tricks.com/specifics-on-css-specificity/

Comment (Ahmedgagan) [reply assets/stylesheets/common/discourse-reactions.scss]: Thanks for the article 😃 added the change in the latest commit

File: assets/stylesheets/common/discourse-reactions.scss [hunk 4]
```diff
@@ -190,84 +190,77 @@ html.discourse-reactions-no-select {
...
-    .d-icon {
-      color: $love;
-    }
+    &:not(.can-toggle-reaction) {
+      .discourse-reactions-reaction-button:hover .reaction-button {
+        background: $primary-low;
+      }
 
-    &.can-toggle-main-reaction {
+      .discourse-reactions-reaction-button button,
```
Comment (jjaffeux) [inline assets/stylesheets/common/discourse-reactions.scss]: what about this one ?

Comment (Ahmedgagan) [reply assets/stylesheets/common/discourse-reactions.scss]: Sorry for this, I added a new commit for this

**Review by jjaffeux — APPROVED** [review]

**Pull request merged**